x-pack/plugin/ccr/qa/chain/build.gradle

```diff
@@ -5,6 +5,7 @@ apply plugin: 'elasticsearch.standalone-test'
 dependencies {
     testCompile project(path: xpackModule('core'), configuration: 'testArtifacts')
     testCompile project(path: xpackModule('ccr'), configuration: 'runtime')
+    testCompile project(':x-pack:plugin:ccr:qa:test-common')
```

Comment:
SQL has this and it sticks it in `qa`.

Reply:
Well, it is in `x-pack/qa/sql` now, but I'm in the process of moving it to `x-pack/plugin/sql/qa`.

Reply:
That is a much better idea than adding another gradle module. So I did this: 633afa77dd1cea40404d8f2de37a62674b39ec4a

Reply:
and here is the second half of the change: 772a4f85695310ad13f93a4283a0a08f9b281f6e